src/views/ConsentContinuationOut.vue

```diff
@@ -576,11 +569,11 @@ export default class ConsentContinuationOut extends Vue {
       this.showErrors = true
       if (!this.detailCommentValid) {
         // Show error message of detail comment text area if invalid
-        this.$refs.detailCommentRef.$refs.textarea.error = true
+        (this.$refs.detailCommentRef.$refs.textarea as any).error = true
```

Comment:
You could also fix this by declaring textarea in DetailComment.vue:
```
  // Refs
  $refs!: {
    textarea: any
  }
```

Reply:
Similar for certifyTextfieldRef below.

Reply:
Tried this and tested, it works! There are always new stuff/ways to learn 👍

Reply:
(Ideally we would declare textarea with a proper type/interface, but that's getting into Vuetify code so I'm OK with this.)

Reply:
Fixed, thanks.